examples/host-function-library/library/json-parser/export/library_plugin.pb.go

```diff
@@ -32,6 +32,7 @@ func (h parserLibrary) ParseJson(ctx context.Context, request *ParseJsonRequest)
 	}
 	ptr, size := wasm.ByteToPtr(buf)
 	ptrSize := _parse_json(ptr, size)
+	wasm.FreePtr(ptr)
```

Comment:
maybe for this cases we could have another variant of the `ByteToPtr` that does not perform any allocation ?

Reply:
Yes, this can be an option for host functions only, but need to validate that's GC working correctly in this case